examples/accesskit.rs

```diff
@@ -114,12 +114,17 @@ impl HelloState {
             }
             _ => unreachable!(),
         };
+        // We have to be slightly less lazy here than we'd like because we can't
```

Comment:
If you put `self.node_classes` in a `RefCell`, then you only need to borrow `self` immutably.

Reply:
The node classes are opaque, so we could indeed use interior mutability, or even an implicit global set of node classes. But I want to push decisions about thread synchronization as far away from the AccessKit core as possible, so both platform adapters and toolkits can use whatever thread synchronization approach (or lack thereof) is best for them, and so the core AccessKit crates can support no-std later.